### ISSUE TITLE
Velero is incompatible with containerd

### ISSUE THREAD

**Issue by mzaneri** [replicatedhq/kURL#576]
If using containerd and velero, the following issue will arise

```
backupstoragelocation.velero.io/default created
./addons/velero/1.2.0/install.sh: line 53: docker: command not found
./addons/velero/1.2.0/install.sh: line 54: docker: command not found
```

This is due to this function, which needs to be updated for when the CRI is containerd:

https://github.com/replicatedhq/kURL/blob/master/addons/velero/1.2.0/install.sh#L46:L58

**Comment by MikaelSmith** [comment]
Looks like this has been fixed.

**Comment by areed** [comment]
Fixed in #589
